### PR TITLE
Fixed Rect getColourConfig() and added a destroy method

### PR DESCRIPTION
**Pull request by jeuxjeux20**
- Added a new `destroy` methods to `BasicFillRect`, `ImageRect` and `TextRect`
- Fixed `getColourConfig()` (`BasicFillRect`, `TextRect`) and `getColourTint()` (`ImageRect`) segfault bug

### CODE REVIEW

File: include/NovelRT.Interop/Graphics/NrtBasicFillRect.h [hunk 1]
```diff
@@ -11,6 +11,9 @@ extern "C"
 {
 #endif
 
+    // destroy
+    NrtResult Nrt_BasicFillRect_destroy(NrtBasicFillRectHandle rect);
```
Comment (RubyNova) [inline include/NovelRT.Interop/Graphics/NrtBasicFillRect.h:15]: Conventionally, destroy methods are the last method of the type to be defined. Can we do that here and in the other places you have added these? And can we please make the cpp files mirror these declarations?

**Review by RubyNova — CHANGES_REQUESTED** [review]
Just a minor change. :)

File: include/NovelRT.Interop/Graphics/NrtBasicFillRect.h (hunk 1 above)
Comment (jeuxjeux20) [reply include/NovelRT.Interop/Graphics/NrtBasicFillRect.h]: Should be done now.

**Review by RubyNova — APPROVED** [review]
LGTM

**Pull request merged**